src/objects/zcl_abapgit_object_aifc.clas.abap

```diff
@@ -279,6 +286,10 @@ CLASS ZCL_ABAPGIT_OBJECT_AIFC IMPLEMENTATION.
     ENDIF.
 
     TRY.
+        IF execute_checks( io_xml = io_xml ) = abap_false.
```

Comment:
if there's just one parameter, please remove the parameter name: execute_checks( io_xml ). There're are few occurances like this. Thanks